app/src/main/java/com/kickstarter/viewmodels/ActivitySampleProjectViewHolderViewModel.kt

```diff
@@ -19,26 +18,31 @@ class ActivitySampleProjectViewHolderViewModel {
         fun bindActivity(): Observable<Activity>
     }
 
-    class ViewModel(@NonNull val environment: Environment) : ActivityViewModel<ActivitySampleProjectViewHolder>(environment), Inputs, Outputs {
+    class ActivitySampleProjectHolderViewModel : ViewModel(), Inputs, Outputs {
```

Comment:
For the ViewHolders we cannot really use `jectpack ViewModel`, the ViewHolderViewModels are tied directly to the Activity/Fragment lifecycle, rather to the Adapter method `onViewDetachedFromWindow`, `onViewAttachedToWindow`.
Plus if you take a look at `ViewModel()` you'll see never to create a viewModel outside of a provider factory 

<img width="692" alt="Screenshot 2023-10-16 at 9 52 46 AM" src="https://github.com/kickstarter/android-oss/assets/4083656/184ec6d8-cce7-4a5f-9816-5bc0cfd60e1d">